src/Common/src/Interop/Unix/System.Native/Interop.FCntl.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal static class Fcntl
+        {
+            internal static readonly bool CanGetSetPipeSz = FcntlCanGetSetPipeSz();
+        }
+
+        [DllImport(Libraries.SystemNative)]
+        [return: MarshalAs(UnmanagedType.I4)]
+        private static extern bool FcntlCanGetSetPipeSz();
+
+        [DllImport(Libraries.SystemNative)]
+        internal static extern unsafe int FcntlGetPipeSz(int fd);
```

Comment:
These need SetLastError=true set on them.  It was a bug that the ```fcntl``` DllImport didn't have them previously.